docs/.gitignore

```diff
@@ -0,0 +1 @@
+book
```

Comment:
is it on purpose that this has its own gitignore? I don't mind, but it's not consistent with other folders. Maybe add this line directly to the root .gitignore?